commercetools-internal-processors/src/main/resources/commercetools-jvm-sdk-processor-templates/commands/updateCommandInterface.hbs

```diff
@@ -45,8 +46,10 @@ public interface {{resourceName}}UpdateCommand extends UpdateCommandDsl<{{resour
         @param updateAction the update to perform
         @return the update command for {{resourceName}}
     */
-    static {{resourceName}}UpdateCommand of(final Versioned<{{resourceName}}> versioned, final UpdateAction<{{resourceName}}> updateAction) {
-        return new {{resourceName}}UpdateCommandImpl(versioned, Collections.singletonList(updateAction));
+    static {{resourceName}}UpdateCommand of(final Versioned<{{resourceName}}> versioned, final UpdateAction<{{resourceName}}> updateAction, final UpdateAction<{{resourceName}}>... updateActions) {
```

Comment:
I would go for a signature that forces you to specify at least one updateAction meaning 
```suggestion
    static {{resourceName}}UpdateCommand of(final Versioned<{{resourceName}}> versioned, final UpdateAction<{{resourceName}}> firstUpdateAction, final UpdateAction<{{resourceName}}>, final UpdateAction<{{resourceName}}>... restOfUpdateActions) {
```

Then Concatenate all update actions in a list before sending it to Commercetools.